qemu/tests/drive_mirror.py

```diff
@@ -122,6 +144,8 @@ def is_steady(self):
         if self.vm.monitor.protocol == "qmp":
             if params.get("check_event", "no") == "yes":
                 ret &= bool(self.vm.monitor.get_event("BLOCK_JOB_READY"))
+        if not ret:
```

Comment:
mass useless information will print during wait for block job ready. if you like please add it after wait_for function.